### PR TITLE
zh-cn(update): sync http_header for the english

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

File: files/zh-cn/web/http/headers/index.md [hunk 1]
```diff
@@ -1,120 +1,408 @@
...
-> **备注：** {{Gecko ("5.0") }}不会发送带有 Keep-Alive 的请求头。以前的版本确实发送过，但格式并不正确，因此决定暂时删除它。{{ httpheader("Connection") }}或{{ httpheader("Proxy-Connection") }}报文首部仍然会发送，带着"keep-alive"的值。
-
-## 更多
-
-[维基百科上对 HTTP 首部的展示](http://en.wikipedia.org/wiki/List_of_HTTP_header_fields)
+{{HTTPSidebar}}
+
+**HTTP 标头**（header）允许客户端和服务器通过 HTTP 请求（request）或者响应（response）传递附加信息。一个 HTTP 标头由它的名称（不区分大小写）后跟随一个冒号（`:`），冒号后跟随它具体的值。该值之前的{{Glossary("Whitespace", "空格")}}会被忽略。
+
+自定义专用的标头之前可以与 `-X` 前缀一起使用，但是这种用法被 IETF 在 2012 年 6 月发布的 [RFC 6648](https://datatracker.ietf.org/doc/html/rfc6648) 中明确弃用，原因是其会在非标准字段成为标准时造成不便；其它的标头在 [IANA 注册表](https://www.iana.org/assignments/message-headers/message-headers.xhtml#perm-headers)中列出，其原始内容在 [RFC 4229](https://datatracker.ietf.org/doc/html/rfc4229) 中定义。此外，IANA 还维护着[被提议的新的 HTTP 标头注册表](https://www.iana.org/assignments/message-headers/message-headers.xhtml#prov-headers)。
```
Comment (fwqaaq) [inline files/zh-cn/web/http/headers/index.md:10]:
```suggestion
自定义专用的标头之前可以与 `-X` 前缀一起使用，但是这种用法被 IETF 在 2012 年 6 月发布的 [RFC 6648](https://datatracker.ietf.org/doc/html/rfc6648) 中明确弃用，原因是其会在非标准字段成为标准时造成不便；其它的标头在 [IANA 注册表](https://www.iana.org/assignments/message-headers/message-headers.xhtml#perm-headers)中列出，其原始内容在 [RFC 4229](https://datatracker.ietf.org/doc/html/rfc4229)中定义。此外，IANA 还维护着[被提议的新的 HTTP 标头注册表](https://www.iana.org/assignments/message-headers/message-headers.xhtml#prov-headers)。
```

File: files/zh-cn/web/http/headers/index.md [hunk 2]
```diff
@@ -1,120 +1,408 @@
...
+- {{ httpheader("Keep-Alive") }}
+- {{ httpheader("Proxy-Authenticate") }}
+- {{ httpheader("Proxy-Authorization") }}
+- {{ httpheader("TE") }}
+- {{ httpheader("Trailer") }}
+- {{ httpheader("Transfer-Encoding") }}
+- {{ httpheader("Upgrade") }}（另见[协议升级机制](/zh-CN/docs/Web/HTTP/Protocol_upgrade_mechanism)）。
+
+- 端到端（End-to-end）标头
+  - : 这类标头*必须*被传输到最终的消息接收者：请求的服务器或者响应的客户端。中间的代理必须重新转发这些未经修改的标头，并且必须换成缓存它们。
```
Comment (fwqaaq) [inline files/zh-cn/web/http/headers/index.md:31]:
```suggestion
  - : 这类标头*必须*被传输到最终的消息接收者：请求的服务器或者响应的客户端。中间的代理必须重新转发这些未经修改的标头，并且必须缓存它们。
```

File: files/zh-cn/web/http/headers/index.md [hunk 3]
```diff
@@ -1,120 +1,408 @@
...
+- {{ httpheader("Proxy-Authorization") }}
+- {{ httpheader("TE") }}
+- {{ httpheader("Trailer") }}
+- {{ httpheader("Transfer-Encoding") }}
+- {{ httpheader("Upgrade") }}（另见[协议升级机制](/zh-CN/docs/Web/HTTP/Protocol_upgrade_mechanism)）。
+
+- 端到端（End-to-end）标头
+  - : 这类标头*必须*被传输到最终的消息接收者：请求的服务器或者响应的客户端。中间的代理必须重新转发这些未经修改的标头，并且必须缓存它们。
+- 逐跳（Hop-by-hop）标头
+  - : 这类标头仅对单词传输连接有意义，并且*不得由*代理重传或者缓存。注意，只能使用 {{httpheader("Connection")}} 标头来设置逐跳标头。
```
Comment (fwqaaq) [inline files/zh-cn/web/http/headers/index.md:33]:
```suggestion
  - : 这类标头仅对单次传输连接有意义，并且*不得由*代理重传或者缓存。注意，只能使用 {{httpheader("Connection")}} 标头来设置逐跳标头。
```

File: files/zh-cn/web/http/headers/index.md [hunk 4]
```diff
@@ -1,120 +1,408 @@
...
+- {{HTTPHeader("Cross-Origin-Embedder-Policy")}}（COEP）
+  - : 允许服务器为给定文档声明嵌入器（embedder）策略。
+- {{HTTPHeader("Cross-Origin-Opener-Policy")}}（COOP）
+  - : 防止其他域打开/控制窗口。
+- {{HTTPHeader("Cross-Origin-Resource-Policy")}}（CORP）
+  - : 防止从其他域读取应用了此标头的资源的响应。
+- {{HTTPHeader("Content-Security-Policy")}}（{{Glossary("CSP")}}）
+  - : 控制允许用户代理能够为给定页面加载哪些资源。
+- {{HTTPHeader("Content-Security-Policy-Report-Only")}}
+  - : 允许 Web 开发人员通过监控而不是强制执行其效果来试验策略。这些违规报告包含通过 HTTP `POST` 请求发送到指定 URI 的 {{Glossary("JSON")}} 文档。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/index.md:281]:
```suggestion
  - : 允许 Web 开发人员通过监控而不是强制执行其效果来试验策略。这些由 {{Glossary("JSON")}} 文档组成的违规报告包含通过 HTTP `POST` 请求发送到指定的 URI 。
```

File: files/zh-cn/web/http/headers/index.md [hunk 5]
```diff
@@ -1,120 +1,408 @@
...
+- {{HTTPHeader("Expect-CT")}}
+  - : 允许网站选择报告和/或执行证书透明度要求，这可以让你注意到该网站使用了欺诈性证书。当站点启用 Expect-CT 标头时，他们将请求 Chrome 检查该站点的任何证书是否出现在公共 CT 日志中。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/index.md:283]:
```suggestion
  - : 允许网站选择报告和/或执行证书透明度要求，这可以让你注意到该网站使用了欺诈性证书。当站点启用 Expect-CT 标头时，他们将请求 Chrome 检查该站点的任何证书是否出现在公共证书透明度日志中。
```

File: files/zh-cn/web/http/headers/index.md [hunk 6]
```diff
@@ -1,120 +1,408 @@
...
+- {{HTTPHeader("Origin-Isolation")}} {{experimental_inline}}
+  - : 提供一种允许 Web 应用程序隔离其来源的机制。
+- {{HTTPHeader("Permissions-Policy")}}
+  - : 提供一种机制来允许和拒绝在网站自己的框架和它嵌入的 {{htmlelement("iframe")}} 中使用浏览器功能。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/index.md:287]:
```suggestion
  - : 提供一种机制来允许和拒绝在网站自己的框架和它嵌入的 {{htmlelement("iframe")}} 中使用浏览器特性。
```

File: files/zh-cn/web/http/headers/index.md [hunk 7]
```diff
@@ -1,120 +1,408 @@
...
+{{Glossary("Fetch metadata request header", "获取元数据请求标头")}}提供有关发起请求的上下文的信息。这允许服务器根据请求的来源和资源将如何使用来决定是否允许请求。
+
+- {{HTTPHeader("Sec-Fetch-Site")}}
+  - : 它是一个请求标头，指示请求发起者的源与其目标源之间的关系。它是一个结构化标头（Structured Header），其值是一个标记，可能的值有 `cross-site`、`same-origin`、`same-site` 和 `none`。
+- {{HTTPHeader("Sec-Fetch-Mode")}}
+  - : 它是一个请求标头，向服务器指示请求的模式。它是一个结构化标头（Structured Header），其值是一个标记，可能的值有 `cors`、`navigate`、`no-cors`、`same-origin` 和 `websocket`。
+- {{HTTPHeader("Sec-Fetch-User")}}
+  - : 它是一个请求标头，指示导航请求是否由用户激活触发。它是一个结构化标头（Structured Header），其值为布尔值，因此可能的值为 `?0` 表​​示 false，`?1` 表示true。
+- {{HTTPHeader("Sec-Fetch-Dest")}}
+  - : 它是一个请求标头，指示请求到服务器的目的地。它是一个结构化标头（Structured Header），其值为具有可能值的标记 `audio`、`audioworklet`、`document`、`embed`、`empty`、`font`、`image`、`manifest`、`object`、`paintworklet`、`report`、`script`、`serviceworker`、`sharedworker`、`style`、`track`、`video`、`worker` 和 `xslt`.
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/index.md:316]:
```suggestion
  - : 它是一个请求标头，指示请求到服务器的目的地。它是一个结构化标头（Structured Header），其值为具有可能值的标记 `audio`、`audioworklet`、`document`、`embed`、`empty`、`font`、`image`、`manifest`、`object`、`paintworklet`、`report`、`script`、`serviceworker`、`sharedworker`、`style`、`track`、`video`、`worker` 和 `xslt`。
```

**Review by yin1999 — COMMENTED** [review]
Thanks. This is really a big contribution. I've list some nits that should be settled before merging

**Comment by fwqaaq** [comment]
❤️

**Review by yin1999 — APPROVED** [review]

**Pull request merged**